index.js

```diff
@@ -144,6 +146,10 @@ const createMainWindow = async () => {
       {
         role: 'help',
         submenu: [
+          {
+            label: 'Reopen welcome tour',
```

Comment:
```suggestion
            label: 'Reopen welcome screens',
```
To more clearly differentiate between the modals and the onboarding tour.